wenab-app/config/BaseApi.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+class BaseApi {
```

Comment:
why is this called BaseApi/what's the purpose of this class?

Reply:
The Wenab client we generate will inherit from this class. This makes it a little cleaner to give the client it's access token since we have to do that for every new instance of the client we instantiate in our TS code.